### ISSUE TITLE
Chore tracking page

### ISSUE THREAD

**Issue by MrJohnZoidberg** [patzly/grocy-android#560]
Where user can select "user who has done the chore"

**Comment by DanielNautre** [comment]
Hi Dominic, could this be set as default so it doesn't need to be specified each time?

**Comment by dominiczedler** [comment]
What do you mean? This chore tracking page does not exist yet. 😄

**Comment by DanielNautre** [comment]
I know 😅 I meant that the user doing the chores should be a setting on the app. But I'm now realizing that this is handled by the API keys. So i guess my comment is not relevant.

**Comment by dominiczedler** [comment]
Yeah I don't know exactly but I think you're right. The additional page would just allow tracking for any user. This will come in the future (I don't know when, sorry). 😃
